src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -6144,6 +6171,23 @@ private boolean containsWarnings( String output )
         }
     }
 
+    /**
+     * Determines whether the specified string is informational output of the Javadoc tool.
+     *
+     * @param str string to check
+     * @return true if informational output, false if not or cannot be determined
+     */
+    private boolean isInfoOutput( String str )
+    {
+        return str == null
+                || str.startsWith( "Loading source file " )
+                || str.startsWith( "Generating " )
+                || str.startsWith( "Constructing Javadoc information" )
+                || str.startsWith( "Building index for " )
+                || str.startsWith( "Building tree for " )
+                || str.startsWith( "Standard Doclet version " );
+    }
```

Comment:
Where do you have this information from? From JDK source?

Reply:
These lines are output of the Javadoc tool.

Reply:
That I understand, but is this list complete? Are those always in English? Have you tried `fr_FR`, `de_AT`?

Reply:
The stdout/err output of the tool appears to be always English. I have tested with a variety of different locales.
The tool has a command line option `-locale name` which affects the generated output (i.e. the html files), not the technical output of the tool itself.

Reply:
I dug further into this. The Javadoc tool has localizations for Japanese and Chinese only it seems.
Found in  AdoptOpenJDK / openjdk-jdk11u: https://github.com/AdoptOpenJDK/openjdk-jdk11u/tree/master/src/jdk.javadoc/share/classes/jdk/javadoc/internal/tool/resources

So for those two my PR won't do anything, the plugin will behave just as before. The implementation is defensive.